config/example/node.yaml

```diff
@@ -99,6 +99,7 @@ storage:
   # note: shard configuration can be omitted for relay node (see `node.relay`)
   shard_pool_size: 15 # size of per-shard worker pools used for PUT operations
   shard_num: 2  # total number of shards
+  shard_ro_error_threshold: 100
```

Comment:
Name is quite telling, but let's still add a verbal explanation (in particular, that with a zero/NaN value, the restriction is not set)

Reply:
Fixed.